packages/playground/src/explorer/components/node_details.vue

```diff
@@ -141,4 +177,7 @@ export default {
 .v-toolbar__content {
   justify-content: end !important;
 }
+.v-toolbar__content {
+  height: 48px !important;
```

Comment:
are we sure that this style doesn't have any unwanted effect in some other place ?

Reply:
It'll effect only on the component